.chloggen/refactor-clickhouse-conn-management.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
It's not a breaking change anymore, right?

Reply:
correct